src/crewai/memory/contextual/contextual_memory.py

```diff
@@ -60,6 +71,19 @@ def _fetch_entity_context(self, query) -> str:
         """
         em_results = self.em.search(query)
         formatted_results = "\n".join(
-            [f"- {result['context']}" for result in em_results]  # type: ignore #  Invalid index type "str" for "str"; expected type "SupportsIndex | slice"
+            [
+                f"- {result['memory'] if self.memory_provider == 'mem0' else result['context']}"
+                for result in em_results
+            ]  # type: ignore #  Invalid index type "str" for "str"; expected type "SupportsIndex | slice"
         )
         return f"Entities:\n{formatted_results}" if em_results else ""
+
+    def _fetch_user_memories(self, query) -> str:
```

Comment:
I think this is never being called?

Reply:
Sorry, there was a pending local commit for this. Pushed now.